content/en/docs/tasks/inject-data-application/define-command-argument-container.md

```diff
@@ -1,66 +1,68 @@
 ---
-title: Define a Command and Arguments for a Container
-content_template: templates/task
+title: Defining a Command and Arguments for a Container
+redirect_from:
+- "/docs/concepts/configuration/container-command-args/"
+- "/docs/concepts/configuration/container-command-arg.html"
 ---
 
-{{% capture overview %}}
```

Comment:
All the shortcodes need to be reverted to the red versions for captures, admonitions, the glossary tooltips, includes, etc, throughout the file.